data/templates/test_report.html

```diff
@@ -26,6 +26,7 @@
   <div class="row">
     <div class="col-md-4">
       <h1>Test Report</h1>
+      <h2>Version: {{version}}</h2>
```

Comment:
Was about to merge, but just noticed this. Now that version is optional, this needs to be `{% if version %}<h2>Version: {{version}}</h2>{% endif %}`, then :+1: